compose/etc/grafana/provisioning/dashboards/Archivematica.json

```diff
@@ -15,6 +15,7 @@
   "editable": true,
   "gnetId": null,
   "graphTooltip": 0,
+  "iteration": 1564762335144,
```

Comment:
What does `iteration` mean in this context?

Reply:
Ahh...I'm guessing that it's a timestamp of when the dashboard was modified or exported? I am doing all of the editing in Grafana and just committing the result of the export. The acutal schema is pretty opaque to me.